src/caf/core/zoning.py

```diff
@@ -41,6 +41,8 @@ class TranslationWeighting(enum.Enum):
     EMPLOYMENT = "employment"
     NO_WEIGHT = "no_weight"
     AVERAGE = "average"
+    POP = "pop"
+    EMP = "emp"
```

Comment:
Ahh yes this was a hot fix because 'population' and 'employment' never actually appear in translation names as weightings so it wasn't finding them. I think I meant to chat to you about this as I think you added this bit initially but I never got round to it.